browser/about_flags.cc

```diff
@@ -583,7 +583,7 @@ constexpr char kBraveAndroidSafeBrowsingDescription[] =
      {kPlaylistFeatureInternalName,                                        \
      flag_descriptions::kPlaylistName,                                     \
      flag_descriptions::kPlaylistDescription,                              \
-     flags_ui::kOsMac | flags_ui::kOsWin | flags_ui::kOsLinux,             \
+     kOsDesktop | kOsAndroid,                                              \
```

Comment:
I think `flags_ui::kOsMac | flags_ui::kOsWin | flags_ui::kOsLinux | kOsAndroid` is more accurate because
`kOsDesktop` includes more - https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/about_flags.cc;l=328?q=kOsDesktop